src/pie-chart.js

```diff
@@ -122,7 +122,7 @@ class PieChart extends PureComponent {
                     style={{ flex: 1 }}
                     onLayout={ event => this._onLayout(event) }
                 >
-                    <Svg style={{ flex: 1 }}>
+                    <Svg style={{ width: this.state.width, height: this.state.height }}>
```

Comment:
width and height are already destructured from state, just do `{{ width, height }}`